content/docs/00-release-notes.md

```diff
@@ -15,6 +15,85 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 
 
+# December xx, 2022 - Release 3.1.0
+
+Palette 3.1 is released with support for AWS GovCloud, FIPS Compliant PXK, and PXK-E Kubernetes versions. This release also features Autoscalers for IaaS clusters. The Palette Developer Experience (PDE) product also contains several enhancements that improve the user experience, such as the ability to pause/resume clusters, new services for app profiles, and more. 
+
+## Palette
+
+### Features: 
+
+* Palette supporters integration with [AWS GovCloud services]() to meet the compliance mandates for safeguarding sensitive data by strengthening identity management, improving cloud visibility, and protecting accounts and workloads to support mission-critical workloads for government customers.
+* [Autoscaling]() capabilities for Palette IaaS clusters to ensure better availability and cost management for dynamic workloads.
+* Palette is now compliant with FIPS compliance and provides a [FIPS-compliant]() version of Kubernetes (PXK and PXK-E).  Palette FIPS support is extended at the Platform Level with the Tenant and Project Scope and Cluster level with FIPS compliant Infra layer cluster profiles.
+* Palette supports tagging and the ability to filter user [access]() and [visibility]() to clusters using tags. 
+* Filter geographically dispersed clusters in the Palette map view and list view using [flexible filters]() to have a granular view of cluster information.
+* Palette supports App Profile versioning. Versioning enables users to create multiple [versions of an App Profile]() within the scope of a single profile name. 
+* Palette supports the [cloning]() of App Profiles across multiple projects. For example, you can clone an app profile created under a specific project to another project within the same tenant. 
+* Palette supports the ability to update an [App Profile]() with the ability to update the versions and manifests. You can also remove and add app tiers.
+* Palette App Mode now supports the use of [containers](). You can specify a single container image when creating an app profile.
+* Palette leverages the [Helm and OCI registries]() to configure the store and service packs. 
+* Palette supports [out-of-the-box]() applications support for services such as Kafka, MySQL, MinIO, NATS, etc., for Palette Dev Engine. These services can be specified when creating an App Profile.
+* Palette allows you to [pause and resume]() virtual clusters that are not in use. This adds significant flexibility in managing the operating costs and optimizing resource management for virtual clusters.
+
+### Enhancements:
+
+* [OS patch reboot]() support allows clusters to reboot to apply system updates if required.
+
+* Palette Tencent Clusters now support using [security groups]() for network isolation and improving security.
+
+* Improved app profile modeling experience.
```

Comment:
What is app profile modeling? Is there a page we can link to? I'm not even sure I know what it is.